capstone/src/main/webapp/youtubeScript.js

```diff
@@ -69,7 +81,7 @@ async function inputCommentsToPerspective(response) {
   }
   const attributeAverages = new Map();
   for (const [attribute, attributeScoresTotal] of attributeTotals) {
-    attributeAverages.set(attribute,attributeScoresTotal / requestedAttributes.length);
+    attributeAverages.set(attribute,attributeScoresTotal / comments.items.length);
```

Comment:
```suggestion
    attributeAverages.set(attribute, attributeScoresTotal / comments.items.length);
```

Reply:
addressed in "address host and intern Daniel's nits"